src/redux/actions/index.js

```diff
@@ -50,6 +33,30 @@ export const findMovieById = (dispatch, id) => {
     })
 }
 
+export const addMovie = (dispatch, movie) => {
+  postMovie(movie)
+    .then(() => dispatch(filterMovies()))
+    .catch(reason => {
+      dispatch(setError(reason));
+    })
+}
+
+export const updateMovie = (dispatch, movie) => {
+  putMovie(movie)
+    .then(() => dispatch(filterMovies()))
+    .catch(reason => {
+      dispatch(setError(reason));
+    })
+}
+
+export const deleteMovie = (dispatch, id) => {
+  deleteMovieById(id)
+    .then(() => dispatch(filterMovies()))
+    .catch(reason => {
+      dispatch(setError(reason));
+    })
+}
```

Comment:
В этих 3 экшинах ни как не обрабатывается респонс от сервера.
```suggestion
export const addMovie = (dispatch, movie) => {
  postMovie(movie)
    .then((data) => dispatch(updateMovies(data)))
    .catch(reason => {
      dispatch(setError(reason));
    })
}

export const updateMovie = (dispatch, movie) => {
  putMovie(movie)
    .then((data) => dispatch(updateMovies(data)))
    .catch(reason => {
      dispatch(setError(reason));
    })
}

export const deleteMovie = (dispatch, id) => {
  deleteMovieById(id)
    .then((data) => dispatch(updateMovies(data)))
    .catch(reason => {
      dispatch(setError(reason));
    })
}
```

Reply:
Added actions for add, update and delete